rfc/graphql-schema/additionalTypeDefs/byPage/userManagement.ts

```diff
@@ -48,6 +48,16 @@ export default /* GraphQL */ `
       SomeChangeCustomerRoleError
     | AnotherChangeCustomerRoleError
 
+  input ChangeCustomerRoleDataInput{
+    customerId: ID!
+    roleId: ID!
+  }
+
+  input ChangeCustomerRoleInput {
+    companyId: ID!
+    data: ChangeCustomerRoleDataInput!
+  }
+
```

Comment:
per our standup discusssion w/ ignacio & victor

customer is the selector id
role is the data we're mutating

```suggestion
  input ChangeCustomerRoleDataInput{
    roleId: ID!
  }

  input ChangeCustomerRoleInput {
    customerId: ID!
    data: ChangeCustomerRoleDataInput!
  }

```